packages/asset-swapper/src/utils/market_operation_utils/fills.ts

```diff
@@ -112,6 +114,7 @@ function dexQuotesToPaths(
     ethToOutputRate: BigNumber,
     fees: FeeSchedule,
 ): Fill[][] {
+    const sourcePathId = hexUtils.random();
```

Comment:
this should be inside the for loop I think

Reply:
I want it to be unique to the whole path. So we can tell which fill belongs to which curve.

Reply:
nvm ur rite

Reply:
yay that fixed my sims